### PR TITLE
fix(rax): component type

### PR DESCRIPTION
**Pull request by SoloJiang**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Review by chenjun1011 — APPROVED** [review]

**Review by imsobear — APPROVED** [review]

**Comment by SoloJiang** [comment]
Ready to merge

**Comment by SoloJiang** [comment]
@andrewbranch Please help me to merge it, thanks

**Comment by andrewbranch** [comment]
1. Please avoid pinging DT maintainers just to get something merged. There are 217 open PRs in this repo, and the vast majority of them are older than yours. There is a process by which we review and merge PRs; you only need to ping a maintainer if there’s an infrastructure problem.
2. You didn’t follow the instructions on the PR template:

    > If changing an existing definition:
    > - [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>

    I cannot possibly review this PR without this.
3. You could have (and still can) self-merge this PR if you update the tests:
    > 😒 Your PR doesn't modify any tests, so it's hard to know what's being fixed, and your changes might regress in the future. Please consider adding tests to cover the change you're making. Including tests allows this PR to be merged by yourself and the owners of this module.
    > 
    > **_This can potentially save days of time for you!_**

**Pull request merged**
